src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,96 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ARRAY_SIZE = 10;
+    private int lastIndexOfArray = 0;
+    private int index = 0;
```

Comment:
```suggestion
    private int size;
```
you can store just 1 value
```size``` is a perfect name, it stores current size, and last available index